src/shogun/base/SGObject.cpp

```diff
@@ -696,15 +692,15 @@ bool CSGObject::equals(CSGObject* other, float64_t accuracy, bool tolerant)
 
 		if (!this_param && other_param)
 		{
-			SG_DEBUG("leaving %s::equals(): parameter %d is NULL where other's "
-					"parameter \"%s\" is not\n", get_name(), other_param->m_name);
+			SG_DEBUG("leaving equals(): parameter %d is NULL where other's "
```

Comment:
This message contains a bug, where is the `%d` populated
(not your fault but you can fix on the fly here)